flax/experimental/nnx/nnx/module.py

```diff
@@ -452,6 +458,52 @@ def modules(self) -> tp.Iterator[tuple[Path, Module]]:
       if isinstance(value, Module):
         yield path, value
 
+  def set_attributes(
+    self, *filters: filterlib.Filter, **attributes: tp.Any
+  ) -> None:
+    """Sets the attributes of nested Modules including the current Module.
+    If the attribute is not found in the Module, it is ignored.
```

Comment:
would you consider a flag that would raise an error if the attribute isn't found?

Reply:
done